dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_impl.cuh

```diff
@@ -114,11 +114,11 @@ void ycbcr_to_rgb_subsampled(ivec2 offset, const Surface2D<uint8_t>& out,
 }
 
 template <bool horz_subsample, bool vert_subsample>
-__global__ void ChromaSubsampleDistortion(const SampleDesc *samples,
+__global__ void ChromaSubsampleDistortion(const ImageDesc *images,
```

Comment:
I'd stay with the old name, because we use the word "sample" to denote an "element of a batch". Also, even in this case, when the elements being processed are clearly images, the SampleDesc is not an ImageDesc, because it describes _two_ images (input and output).

Reply:
Will drop it